app/ai-gateway/monitor-ai-llm-metrics.md

```diff
@@ -110,6 +121,18 @@ ai_cache_embeddings_latency{ai_provider="provider1",ai_model="model1",cache_stat
 # HELP ai_llm_provider_latency AI cache latencies per ai_provider/database in Kong
 # TYPE ai_llm_provider_latency bucket
 ai_llm_provider_latency{ai_provider="provider1",ai_model="model1",cache_status="hit",vector_db="redis",embeddings_provider="openai",embeddings_model="text-embedding-3-large",Workspace="workspace1",le="+Inf"} 2
+
+# HELP kong_ai_mcp_response_body_size_bytes MCP server response body sizes in bytes
+# TYPE kong_ai_mcp_response_body_size_bytes histogram
+kong_ai_mcp_response_body_size_bytes_bucket{service="svc1",route="route1",method="POST",workspace="workspace1",tool_name="tool1",le="+Inf"} 1
```

Comment:
```suggestion
kong_ai_mcp_response_body_size_bytes_bucket{service="svc1",route="route1",method="tools/call",workspace="workspace1",tool_name="tool1",le="+Inf"} 1
```

I changed the method because:
1. the method is RPC method name, not HTTP method
2. only `tools/call` method has tool_name argument